plugins/polio/js/src/components/Inputs/ScopeInput.tsx

```diff
@@ -355,9 +456,51 @@ export const ScopeInput: FunctionComponent<FieldProps<Scope[], Values>> = ({
             </Tooltip>
         );
     };
+    // Launch the searchdistrictByname from parent component
+    useEffect(() => {
+        searchDistricts.current = searchDistrictByName;
```

Comment:
You're mutating a prop directly, which is not recommended. See the [es-lint rule](https://eslint.org/docs/latest/rules/no-param-reassign)